schema/deploy/computed_columns/contact_pending_form_change.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy cif:computed_columns/contact_pending_form_change to pg
+
+begin;
+
+create function cif.contact_pending_form_change(cif.contact) returns cif.form_change as $$
+  select * from cif.form_change
+  where form_data_schema_name = 'cif'
+  and form_data_table_name = 'contact'
+  and form_data_record_id = $1.id
+  and change_status = 'pending';
+$$ language sql stable;
+
```

Comment:
this will probably blow up once we allow concurrent editing, we'll have to filter by user in session